### ISSUE TITLE
Azure Optimization Engine and multi tenancy

### ISSUE THREAD

**Issue by ChristianLloydGrannes** [microsoft/finops-toolkit#1036]
## 📝 Scenario
As a consumer of the Azure Optimization Engine I need to connect to several Azure tenants to get all information into in single pane of glass. VM right-size augmented recommendation requires us to connect to Log Analytics workspaces. This is currently onlys supported towards a singkle tenant. (The tenant you deploy the AOE)

## 💎 Solution
Enable possibility to add additional log analytics workspaces from several Azure tenants to AOE. Similar way as we connect several workspaces to the existing solution from same Azure tenant.

## 📋 Tasks
```[tasklist]
### Required tasks
- [ ] TODO: Enable possibility to add Log Analytics Workspace from another tenant for the VM right-size augmented recommendation to work.
```

## 🙋‍♀️ Ask for the community

We could use your help:
1. Please vote this issue up (👍) to prioritize it.
2. Leave comments to help us solidify the vision.

**Comment by helderpinto** [comment]
Thank you for submitting this feature request, @ChristianLloydGrannes. 

[Cross-tenant workspace queries](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/cross-workspace-query) are supported via Azure Lighthouse. This is an investigation path we will follow to address your request. Alternatively, we can also use the secondary tenant Service Principal to collect Perf metrics, but this approach is more convoluted. We cannot share an ETA for this feature, but this is something we will look at in the next releases.

**Comment by arthurclares** [comment]
@helderpinto I'm adding this to November's milestone, but feel free to push it to December if needed.

**Comment by helderpinto** [comment]
@ChristianLloydGrannes, a [PR](https://github.com/microsoft/finops-toolkit/pull/1196) was proposed to address your issue. After following the Azure Lighthouse guidance included in the AOE documentation, you'll just have to add workspace IDs from the secondary tenant to the `AzureOptimization_RightSizeAdditionalPerfWorkspaces` Automation Account variable. This issue will be automatically closed once the PR is merged into the `dev` branch.
